modules/ipi-install-diagnosing-duplicate-mac-address.adoc

```diff
@@ -0,0 +1,48 @@
+[id="ipi-install-diagnosing-duplicate-mac-address_{context}"]
+= Diagnosing a duplicate MAC address when provisioning a new host in the cluster
+
+[role="_abstract"]
+If the MAC address of an existing bare-metal node in the cluster matches the MAC address of a bare-metal host you are attempting to add to the cluster, the Ironic Bare Metal Operator associates the bare-metal host with the existing node. If the host enrollment, inspection, cleaning, or other Ironic steps fail, the `metal3-baremetal-operator` retries the installation continuously. The Operator displays a registration error for the failed bare-metal host. You can diagnose a duplicate MAC address by examining the `metal3-baremetal-operator` log.
+
+.Prerequisites
+
+* Install a {product-title} cluster on bare metal.
+* Install the {product-title} CLI `oc`.
+* Log in as a user with `cluster-admin` privileges.
+
+.Procedure
+
+To determine whether a bare-metal host that fails provisioning has a duplicate MAC address, do the following:
+
+. Get the pods running in the `openshift-machine-api` namespace:
++
+[source,terminal]
+----
+$ oc get pods -n openshift-machine-api
+----
++
+.Example output
+[source,terminal]
+----
+NAME                                           READY   STATUS    RESTARTS   AGE
+cluster-autoscaler-operator-77c59896d5-vrc5l   2/2     Running   1          20d
+machine-api-controllers-69d94b9d85-qtrnf       7/7     Running   1          20d
+machine-api-operator-6899f4c5d6-n9zh2          2/2     Running   1          20d
+metal3-7cbcc4b66b-2rkrl                        8/8     Running   0          20d
+----
+
+. Copy the full name of the metal3 pod, for example `metal3-7cbcc4b66b-2rkrl`.
```

Comment:
Can you check with QE if you can replace steps 1-3 with the "metal3" equivalent of the following?

`oc logs -n openshift-machine-api deployment/cluster-baremetal-operator cluster-baremetal-operator`

It's just simpler for our customers and a little more polished.

If we cannot get QE review, maybe just remove this step...I can't put my finger on exactly what is so eye-catching _to me_ about it:
* "metal3 pod" seems like it should be either the name of the app, Operator,...not sure.
* What to specify in the next step is pretty straightforward.
* It's not really a step, _to me_.  It could be just a para, like maybe "\n+The name of the <whatever> pod is needed in the next step."

I'd just work really hard at making 1-3 into a single step.

Reply:
This MAC address error is now surfaced by `oc get bmh -n openshift-machine-api` for 4.7.

Reply:
So I've made your changes, thanks.